### ISSUE TITLE
count-html-elements count-polaris-components returning different numbers to coverage-tsx

### ISSUE THREAD

**Issue by alex-page** [Shopify/polaris#7685]

**Comment by samrose3** [comment]
On the last statement:

> `coverage-tsx`: Gets both the HTML elements and Polaris components

I think it is more accurate to say

> `coverage-tsx`: Gets the all the lines associated with HTML elements and Polaris components

Should we revise the `coverage-tsx` calculation to be HTML element count vs Polaris components count?

**Comment by alex-page** [comment]
> Should we revise the coverage-tsx calculation to be HTML element count vs Polaris components count?

Yeah I was thinking this. The numbers should correlate. I don't know how big of a change this would be though.
